web/client/components/geostory/builder/Settings.jsx

```diff
@@ -63,6 +63,16 @@ export default ({
                 disableShadow
                 onChange={(general) => onUpdateSettings("theme", { ...settings?.theme, general })}/>
         </FormGroup>
+        <FormGroup>
+            <div style={{ marginBottom: "10px"}}>
+                <ControlLabel><Message msgId="geostory.builder.settings.overlay"/></ControlLabel>
+            </div>
+            <CustomThemePicker
+                themeStyle={settings?.theme?.overlay}
+                placement="right"
+                disableBackgroundAlpha
```

Comment:
Just noticed we should enable the background alpha value for overlay so we should remove disableBackgroundAlpha.